### PR TITLE
Fix Page indexing issues

### PR DESCRIPTION
**Pull request by iArchitSharma**
image

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: gatsby-node.js [hunk 1]
```diff
@@ -76,6 +76,29 @@ exports.createPages = async ({ actions, graphql, reporter }) => {
...
+  createRedirect({ fromPath: "/cloud-native-management/meshery/integrations/aws-node-termination-handler-2/", toPath: "/cloud-native-management/meshery/integrations/aws-node-termination-handler", redirectInBrowser: true, isPermanent: true });
+  createRedirect({ fromPath: "/cloud-native-management/meshery/integrations/service-mesh-interface-smi.html", toPath: "/projects/service-mesh-interface-conformance", redirectInBrowser: true, isPermanent: true });
+  createRedirect({ fromPath: "/cloud-native-management/gcp-diagram", toPath: "/cloud-native-management/generate-gcp-architecture-diagram", redirectInBrowser: true, isPermanent: true });
+  createRedirect({ fromPath: "/whiteboard", toPath: "/solutions/architecture-diagram", redirectInBrowser: true, isPermanent: true });
+  createRedirect({ fromPath: "/comments", toPath: "/cloud-native-management/meshmap/collaborate/peer-reviews", redirectInBrowser: true, isPermanent: true });
+  createRedirect({ fromPath: "/careers/company/legal/terms-of-service", toPath: "/company/legal/terms-of-service", redirectInBrowser: true, isPermanent: true });
```
Comment (leecalcote) [inline gatsby-node.js:85]: Where is this invalid link coming from? Can we correct this at source instead of redirecting?

Comment (iArchitSharma) [reply gatsby-node.js:85]: @leecalcote, I searched for each of these links in our codebase, but none of them exist within our codebase. Therefore, these are external links from other websites

**Comment by vishalvivekm** [comment]
Let's discuss this on websites call. Add this as an agenda item to the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#), if you would :)

**Review by leecalcote — APPROVED** [review]
Thank you, @iArchitSharma 👍

**Review by leecalcote — COMMENTED** [review]
@iArchitSharma will you please have another look at this site build error:

```
error "gatsby-plugin-meta-redirect" threw an error while running the onPostBuild lifecycle:

ENOTDIR: not a directory, open '/home/runner/work/layer5/layer5/public/products.html/index.html'
```

**Comment by sudhanshutech** [comment]
@iArchitSharma is it completed? what about failing check?

**Pull request closed without merging**